etna/datasets/internal_datasets.py

```diff
@@ -510,4 +550,9 @@ def get_m3_dataset(dataset_dir: Path, dataset_freq: str) -> None:
         "freq": "H",
         "parts": ("train", "test", "full"),
     },
+    "traffic_2015_hourly": {
```

Comment:
Should we clarify that it is hourly?

Reply:
Yes, If we suddenly want to add another frequency, this name will help us.